### DISCUSSION TITLE
API to consume by trackers or any other way of syncing with Ryot?

### DISCUSSION THREAD

**Discussion by alexanderadam** [SamTV12345/PodFetch#709] [Q&A]
I asked this [at the Riot repo](https://github.com/IgnisDa/ryot/issues/630) but now I was thinking whether it's even possible to integrate these two applications since I never read anything about an API.

Or do you have any other idea to make these two interact?

**Comment by SamTV12345** [comment]
I cold put something together. For now there is only the http API, websockets and the telegram api.

**Reply by alexanderadam** [reply]
Maybe the existing API might already work?
Is this documented anywhere?

**Reply by SamTV12345** [reply]
Sure you can find the swagger api path when you start podfetch in the terminal logs
